src/cdk/platform/platform.ts

```diff
@@ -30,12 +30,10 @@ try {
  */
 @Injectable({providedIn: 'root'})
 export class Platform {
-  /**
-   * Whether the Angular application is being rendered in the browser.
-   * We want to use the Angular platform check because if the Document is shimmed
-   * without the navigator, the following checks will fail. This is preferred because
-   * sometimes the Document may be shimmed without the user's knowledge or intention
-   */
+  /** Whether the Angular application is being rendered in the browser. */
```

Comment:
Can you keep the JsDoc directly above the field with the implementation comment above?

Reply:
Done.